minishift/README.md

```diff
@@ -0,0 +1,123 @@
+## Running Git Service on Minishift
+
+These instructions will help you run the service using MiniShift.
+
+### Prerequisites
+
+[MiniShift v1.27.0](https://docs.okd.io/latest/minishift/getting-started/installing.html)
+
+[oc 3.11.0](https://docs.okd.io/latest/cli_reference/get_started_cli.html#installing-the-cli)
+
+[KVM Hypervisor](https://www.linux-kvm.org/page/Downloads)
+
+#### Install Minishift
+
+Make sure you have all prerequisites installed. Please check the list [here](https://docs.openshift.org/latest/minishift/getting-started/installing.html#install-prerequisites)
+
+Download and put `minishift` in your $PATH by following steps [here](https://docs.openshift.org/latest/minishift/getting-started/installing.html#manually)
+
+Verify installation by running following command, you should get version number.
+```bash
+minishift version
+```
+
+#### Install oc
```

Comment:
not needed thanks to `eval $(minishift oc-env)` for those you use Minishift

Reply:
Changed in https://github.com/redhat-developer/git-service/pull/2/commits/7e27a924e0dd4aa711dbcf9f07e1b7f858a5fa4c

Reply:
thanks